dashboard/app/dsl/level_group_dsl.rb

```diff
@@ -27,6 +28,19 @@ def page
   end
 
   def level(name)
+    # Ensure level name hasn't already been used.
+    if @level_names.include? name
```

Comment:
@levels is cleared upon each page, but we want to make sure levels are unique for the entire level group.